lib/Serialization/ModuleFormat.h

```diff
@@ -58,7 +58,7 @@ const uint16_t SWIFTMODULE_VERSION_MAJOR = 0;
 /// describe what change you made. The content of this comment isn't important;
 /// it just ensures a conflict if two people change the module format.
 /// Don't worry about adhering to the 80-column limit for this line.
-const uint16_t SWIFTMODULE_VERSION_MINOR = 820; // begin_borrow/move_value [var_decl]
+const uint16_t SWIFTMODULE_VERSION_MINOR = 821; // add -enable-ossa-modules to .swiftinterface file
```

Comment:
Why does this require a module format version change? If the compiler already supports serializing and deserializing binary modules OSSA information either present or not present, then this change shouldn't have a meaningful effect on deserialization.

Reply:
Okay, I can remove it. I thought changing the stdlib's .swiftinterface in any way also needed a version update.

Reply:
It's of course fairly harmless to bump the module version, but I wanted to understand if there was something I was missing. I don't know of any reason that changes to the stdlib interface in general would require a module version bump.